adapt-service-without-webapi/pom.xml

```diff
@@ -28,67 +27,92 @@
       <artifactId>knotx-adapter-common</artifactId>
       <version>${knotx.version}</version>
     </dependency>
+    <dependency>
+      <groupId>io.knotx</groupId>
+      <artifactId>knotx-knot-handlebars</artifactId>
+      <version>${knotx.version}</version>
+    </dependency>
 
     <dependency>
       <groupId>io.vertx</groupId>
       <artifactId>vertx-jdbc-client</artifactId>
-      <version>3.4.1</version>
+      <version>3.4.2</version>
```

Comment:
Why do you use 3.4.2 version? It is compatible with Vert.x 3.4.1? We plan to move to newer version in future...